api/v1alpha1/kluctldeployment_types.go

```diff
@@ -486,6 +487,14 @@ func SetDeployResult(k *KluctlDeployment, revision string, result *types.Command
 			k.Status.LastDeployResult.RawResult = &raw
 		}
 	}
+	numberOfChangedObjects := len(k.Status.LastDeployResult.ParseResult().ChangedObjects)
```

Comment:
No need to use `ParseResult` here. The `result` parameter is what you need to use here. Maybe This should also be put into a separate function, e.g. `SendCommandResultMetrics()`. This would also remove the need to have the same code in all 3 SetXXXResult functions.

Reply:
I assume that it would also be better to call this new function only from `preparedTarget.handleCommandResult`

Reply:
Much simpler and cleaner now :+1: Thank you :smiley_cat: